CHANGELOG.md

```diff
@@ -6,9 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.1.0]
+## [3.0.0]
 ### Added
-- `insar_tops_multi_burst` workflow for processing multiple bursts as one SLC.
+- The ability for the `insar_tops_burst` workflow to support processing multiple bursts as one SLC.
+- Function to check if the reference granules are older than the secondary granules.
```

Comment:
Something along these lines, to make it clear that an error will be raised? (And this should probably go under `Changed`, not `Added`.)
```suggestion
- The `insar_tops_burst` workflow now exits with an error message if the reference granules are not older than the secondary granules, rather than silently re-ordering the granules.
```